libs/user/user.ts

```diff
@@ -119,15 +126,38 @@ class User {
   }
 
   /**
-   * Retrieve Nightfall mnemonic - remember you must keep it private
+   * Allow user to retrieve the Nightfall Mnemonic  - Keep this private
    *
    * @method getNightfallMnemonic
-   * @return {string} Nightfall mnemonic
+   * @return {String} Nightfall mnemonic
    */
   getNightfallMnemonic(): string {
     return this.nightfallMnemonic;
   }
 
+  /**
+   * Allow user to retrieve Nightfall Layer2 address
+   *
+   * @method getNightfallAddress
+   * @returns {String} Nightfall Layer2 address
+   */
+  getNightfallAddress(): string {
+    return this.zkpKeys?.compressedZkpPublicKey;
+  }
+
+  /**
+   * Allow user to make an ERC20 deposit to Nightfall
+   *
+   * @async
+   * @method makeDeposit
+   * @param {UserMakeDeposit} options
+   * @param {String} options.tokenAddress
+   * @param {String} options.tokenStandard
+   * @param {String} options.value
+   * @param {String} [options.feeWei]
+   * @throws {Error} Argument is not valid bip39 mnemonic
```

Comment:
- Do brackets mean "optional" param? Is this standard?? (eg in JSDocs)
- Not sure the error belongs here? :thinking: I think this prob belongs to `UserFactory`. It's not too important anyways since I'm reviewing returns and error handling, so we can come back to this later.

Let me know

Reply:
Yes having brackets mean it is optional in JSDocs. 
Error was here, we can drop it, i also don't think it should belong here, since it is thrown upon `User` creation.
@imagobea